### PR TITLE
Render product info

### PR DESCRIPTION
**Pull request by tomki1**
- makes axios get request in App.jsx to get one product and passes it to Overview component
- makes axios get request in Overview.jsx to get styles of the current product and passes it to child components
- divided the Overview into halves: Image Gallery on the left and Product Information on the right.
- displays the product title and product description in the Product Information
- displays the first thumbnail of each style of the product in rows of 4.  Crops each style thumbnail to ensure they are the same size without skewing the image
- added '--coverage' to jest test script in package.json
- added Overview tests
- added example.config.js.  Copy its contents into a config.js and input your API token

I will look into changing my css to instead use styled-components.

### CODE REVIEW

File: .gitignore
Comment (LLaxt) [inline .gitignore:1]: add bundle.js files to .gitignore and remove the two bundle files from pull

File: client/dist/styles.css
Comment (LLaxt) [inline client/dist/styles.css:1]: this looks good for now - let's sync up on future changes to the main styles file

File: client/src/components/Overview/ProductInformation.test.jsx
Comment (LLaxt) [inline client/src/components/Overview/ProductInformation.test.jsx:1]: nice work!

**Review by LLaxt — APPROVED** [review]

**Comment by tomki1** [comment]
Closing PR to make the get request in the client side go to the server side and then to the current host of the Atelier API as discussed on 8/13/23 on Slack

**Pull request closed without merging**